fastlane/docs/Actions.md

```diff
@@ -1603,6 +1603,22 @@ git_commit(path: ["./version.txt", "./changelog.txt"]
         message: "Version Bump")
 ```
 
+
+### git_status
+
+To simply get the status of one file or directory
+
+```ruby
+git_status(path: "./version.txt")
+```
+
+To get the status of several files or directories use
+
+```ruby
+git_status(path: ["./version.txt", "./changelog.txt"])
```

Comment:
It is not clear on what this action returns by reading the sample calls. Could you add it here?